README.md

```diff
@@ -37,411 +37,41 @@ Special acknowledgment to the authors of [Playwright](https://playwright.dev/) a
-| [Request](https://playwright.dev/docs/api/class-request) | :white_check_mark: | [`failure()`](https://playwright.dev/docs/api/class-request#request-failure), [`postDataJSON()`](https://playwright.dev/docs/api/class-request#request-post-data-json), [`redirectFrom()`](https://playwright.dev/docs/api/class-request#request-redirected-from), [`redirectTo()`](https://playwright.dev/docs/api/class-request#request-redirected-to) |
-| [Response](https://playwright.dev/docs/api/class-response) | :white_check_mark: | [`finished()`](https://playwright.dev/docs/api/class-response#response-finished) |
-| [Route](https://playwright.dev/docs/api/class-route) | :warning: | All |
-| [Selectors](https://playwright.dev/docs/api/class-selectors) | :warning: | All |
-| [Touchscreen](https://playwright.dev/docs/api/class-touchscreen) | :white_check_mark: | - |
-| [Tracing](https://playwright.dev/docs/api/class-tracing) | :warning: | All |
-| [Video](https://playwright.dev/docs/api/class-video) | :warning: | All |
-| [WebSocket](https://playwright.dev/docs/api/class-websocket) | :warning: | All |
-| [Worker](https://playwright.dev/docs/api/class-worker) | :warning: | All |
+To find out more about xk6-browser or browser testing in general, check out the following links. 
 
+**Note:** since our API is still transitioning, some of the code snippets from the links below might be inaccurate, though the concepts should still be correct. For the latest, refer to our up-to-date [API documentation](https://k6.io/docs/javascript-api/xk6-browser/api/).
 
-Support
--------
+- [Introducing browser automation and end-to-end web testing with k6](https://k6.io/blog/announcing-xk6-browser-testing/) - A blog post from our CEO, Robin Gustafsson, where he introduced xk6-browser.
```

Comment:
@mdcruz I haven't reviewed the content of all the links but I imagine most of them describe the same content: what is xk6-browser + getting started. 

I suggest referencing fewer options.

Reply:
I have removed a few of the duplicate ones.